101-asp-net-app-service/azuredeploy.json

```diff
@@ -0,0 +1,134 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "type": "string",
+            "metadata":{

```

Comment:
add a defaultValue using uniqueString to provide a globally unique default

Reply:
The uniqueString function is okay for an Azure CLI deployment through Powershell or CMD but for the One Button Deployment, it doesn't generate a random name. It shows that like the function and for the location too. 
![image](https://user-images.githubusercontent.com/56584265/68195167-a93d2c00-ffb6-11e9-981e-65099540aabd.png)